### PR TITLE
[REF] auto-fix conventions (with manual fix)

### PR DESCRIPTION
**Pull request by mdietrichc2c**

### CODE REVIEW

File: account_easy_reconcile/easy_reconcile.py [hunk 1]
```diff
@@ -27,6 +27,7 @@
 
 
 class EasyReconcileOptions(orm.AbstractModel):
+
     """Options of a reconciliation profile
```
Comment (bwrsandman) [inline account_easy_reconcile/easy_reconcile.py:30]: :-1: Is the newline before a doctring a positive change to style?

File: account_easy_reconcile/easy_reconcile_history.py [hunk 2]
```diff
@@ -24,6 +24,7 @@
 
 
 class EasyReconcileHistory(orm.Model):
+
     """ Store an history of the runs per profile
     Each history stores the list of reconciliations done"""
```
Comment (bwrsandman) [inline account_easy_reconcile/easy_reconcile_history.py:28]: :-1: Is the newline before a doctring a positive change to style?

File: account_statement_bankaccount_completion/statement.py [hunk 3]
```diff
@@ -28,6 +28,7 @@
 
 
 class AccountStatementCompletionRule(Model):
+
     """Add a rule based on transaction ID"""
```
Comment (bwrsandman) [inline account_statement_bankaccount_completion/statement.py:31]: :-1: Is the newline before a doctring a positive change to style?

File: account_statement_base_completion/partner.py [hunk 4]
```diff
@@ -23,6 +23,7 @@
 
 
 class ResPartner(orm.Model):
+
     """Add a bank label on the partner so that we can use it to match
     this partner when we found this in a statement line.
     """
```
Comment (bwrsandman) [inline account_statement_base_completion/partner.py:28]: :-1: Is the newline before a doctring a positive change to style?

File: account_statement_base_completion/statement.py [hunk 5]
```diff
@@ -40,6 +40,7 @@
 
 
 class ErrorTooManyPartner(Exception):
+
     """ New Exception definition that is raised when more than one partner is
     matched by the completion rule.
     """
```
Comment (bwrsandman) [inline account_statement_base_completion/statement.py:45]: :-1: Is the newline before a doctring a positive change to style?

File: account_statement_base_completion/statement.py [hunk 6]
```diff
@@ -55,6 +56,7 @@ def __repr__(self):
 
 
 class AccountStatementProfil(orm.Model):
+
     """Extend the class to add rules per profile that will match at least the
     partner, but it could also be used to match other values as well.
     """
```
Comment (bwrsandman) [inline account_statement_base_completion/statement.py:60]: :-1: Is the newline before a doctring a positive change to style?

File: account_statement_base_completion/statement.py [hunk 7]
```diff
@@ -112,6 +114,7 @@ def _find_values_from_rules(self, cr, uid, calls, line, context=None):
 
 
 class AccountStatementCompletionRule(orm.Model):
+
     """This will represent all the completion method that we can have to
     fullfill the bank statement lines. You'll be able to extend them in you own
     module and choose those to apply for every statement profile.
```
Comment (bwrsandman) [inline account_statement_base_completion/statement.py:117]: :-1: Is the newline before a doctring a positive change to style?

File: account_statement_base_completion/statement.py [hunk 8]
```diff
@@ -381,6 +384,7 @@ def button_confirm_bank(self, cr, uid, ids, context=None):
 
 
 class AccountStatementLine(orm.Model):
+
     """
     Add sparse field on the statement line to allow to store all the bank infos
     that are given by a bank/office. You can then add you own in your module.
```
Comment (bwrsandman) [inline account_statement_base_completion/statement.py:386]: :-1: Is the newline before a doctring a positive change to style?

File: account_statement_base_completion/statement.py [hunk 9]
```diff
@@ -547,6 +551,7 @@ def _update_line(self, cr, uid, vals, context=None):
 
 
 class AccountBankStatement(orm.Model):
+
     """We add a basic button and stuff to support the auto-completion
     of the bank statement once line have been imported or manually fullfill.
     """
```
Comment (bwrsandman) [inline account_statement_base_completion/statement.py:552]: :-1: Is the newline before a doctring a positive change to style?

File: account_statement_base_import/parser/file_parser.py [hunk 10]
```diff
@@ -36,6 +36,7 @@ def float_or_zero(val):
 
 
 class FileParser(BankStatementImportParser):
+
     """Generic abstract class for defining parser for .csv, .xls or .xlsx file
     format.
     """
```
Comment (bwrsandman) [inline account_statement_base_import/parser/file_parser.py:41]: :-1: Is the newline before a doctring a positive change to style?

File: account_statement_base_import/parser/generic_file_parser.py [hunk 11]
```diff
@@ -27,6 +27,7 @@
 
 
 class GenericFileParser(FileParser):
+
     """Standard parser that use a define format in csv or xls to import into a
     bank statement. This is mostely an example of how to proceed to create a
     new parser, but will also be useful as it allow to import a basic flat
```
Comment (bwrsandman) [inline account_statement_base_import/parser/generic_file_parser.py:32]: :-1: Is the newline before a doctring a positive change to style?

File: account_statement_completion_label/statement.py [hunk 12]
```diff
@@ -28,6 +28,7 @@
 
 
 class ErrorTooManyLabel(Exception):
+
     """New Exception definition that is raised when more than one label is
     matched by the completion rule.
     """
```
Comment (bwrsandman) [inline account_statement_completion_label/statement.py:33]: :-1: Is the newline before a doctring a positive change to style?

File: account_statement_completion_label/statement.py [hunk 13]
```diff
@@ -40,6 +41,7 @@ def __str__(self):
 
 
 class AccountBankSatement(orm.Model):
+
     """We add a basic button and stuff to support the auto-completion
     of the bank statement once line have been imported or manually fullfill.
     """
```
Comment (bwrsandman) [inline account_statement_completion_label/statement.py:45]: :-1: Is the newline before a doctring a positive change to style?

File: account_statement_completion_label/statement.py [hunk 14]
```diff
@@ -114,6 +116,7 @@ def get_from_label_and_partner_field(self, cr, uid, st_line, context=None):
 
 
 class AccountStatementLabel(orm.Model):
+
     """Create a new class to map an account statement label to a partner
     and a specific account
     """
```
Comment (bwrsandman) [inline account_statement_completion_label/statement.py:119]: :-1: Is the newline before a doctring a positive change to style?

File: account_statement_ext/statement.py [hunk 15]
```diff
@@ -39,6 +39,7 @@ def fixed_write(self, cr, uid, ids, vals, context=None):
 
 
 class AccountStatementProfile(orm.Model):
+
     """A Profile will contain all infos related to the type of
     bank statement, and related generated entries. It defines the
     journal to use, the partner and commision account and so on.
```
Comment (bwrsandman) [inline account_statement_ext/statement.py:44]: :-1: Is the newline before a doctring a positive change to style?

File: account_statement_ext/statement.py [hunk 16]
```diff
@@ -113,6 +114,7 @@ def _check_partner(self, cr, uid, ids, context=None):
 
 
 class AccountBankStatement(orm.Model):
+
     """We improve the bank statement class mostly for :
     - Removing the period and compute it from the date of each line.
     - Allow to remove the balance check depending on the chosen profile
```
Comment (bwrsandman) [inline account_statement_ext/statement.py:118]: :-1: Is the newline before a doctring a positive change to style?

File: account_statement_ext/statement.py [hunk 17]
```diff
@@ -591,6 +593,7 @@ def onchange_imp_config_id(self, cr, uid, ids, profile_id, context=None):
 
 
 class AccountBankStatementLine(orm.Model):
+
     """Override to compute the period from the date of the line, add a method
     to retrieve the values for a line from the profile. Override the on_change
     method to take care of the profile when fullfilling the bank statement
```
Comment (bwrsandman) [inline account_statement_ext/statement.py:596]: :-1: Is the newline before a doctring a positive change to style?

File: account_statement_ofx_import/parser/ofx_parser.py [hunk 18]
```diff
@@ -31,6 +31,7 @@
 
 
 class OfxParser(BankStatementImportParser):
+
     """Class for defining parser for OFX file format."""
```
Comment (bwrsandman) [inline account_statement_ofx_import/parser/ofx_parser.py:34]: :-1: Is the newline before a doctring a positive change to style?

File: account_statement_transactionid_import/parser/transactionid_file_parser.py [hunk 19]
```diff
@@ -25,6 +25,7 @@
 
 
 class TransactionIDFileParser(FileParser):
+
     """TransactionID parser that use a define format in csv or xls to import
     bank statement.
     """
```
Comment (bwrsandman) [inline account_statement_transactionid_import/parser/transactionid_file_parser.py:30]: :-1: Is the newline before a doctring a positive change to style?

File: account_statement_base_import/wizard/import_statement.py [hunk 20]
```diff
@@ -32,7 +32,7 @@
 class CreditPartnerStatementImporter(orm.TransientModel):
     _name = "credit.statement.import"
 
-    def default_get(self, cr, uid, fields, context=None):
+    def default_get(self, cr, uid, field_list, context=None):
```
Comment (bwrsandman) [inline account_statement_base_import/wizard/import_statement.py]: :-1:

**Comment by mdietrichc2c** [comment]
@bwrsandman thanks, missed it (along with a few other OpenERP methods that should not be redefined).

**Comment by pedrobaeza** [comment]
:+1:

**Comment by gurneyalex** [comment]
:+1:

**Comment by guewen** [comment]
:+1: 
Thanks

**Pull request merged**